src/main/starlark/release_archive/repository.bzl

```diff
@@ -65,6 +65,12 @@ def archive_repository_implementation(repository_ctx):
             archive = release_artifact,
         )
 
+        # update release when the contents change.
+        return [
+            repository_ctx.path("../../%s" % release_archive.workspace_root),
```

Comment:
Well this seems important. I'm a little shocked it didn't have this before.

Reply:
My bad. Didn't think that i'd need to tell it that the change happened.